pkg/vmimage/data/etc/default/grub

```diff
@@ -0,0 +1,8 @@
+GRUB_TIMEOUT=5
+GRUB_DISTRIBUTOR="$(sed 's, release .*$,,g' /etc/system-release)"
+GRUB_DEFAULT=saved
+GRUB_DISABLE_SUBMENU=true
+GRUB_TERMINAL="serial console"
+GRUB_SERIAL_COMMAND="serial --speed=115200"
+GRUB_CMDLINE_LINUX="console=ttyS0,115200n8 crashkernel=128M"
```

Comment:
I think by removing the line `%addon com_redhat_kdump --disable` in script.sh you may not need this file to be checked in? ptal.

Reply:
yep that was right